src/gui/torrentcategorydialog.h

```diff
@@ -29,6 +29,7 @@
 #pragma once
 
 #include <QDialog>
+#include <QPushButton>
```

Comment:
It shouldn't be here too...

Reply:
@glassez Without that I just confirmed that build fails due to `invalid use of incomplete type 'class QPushButton'`. `setEnabled` call cannot be made. Should I move the `#include` to the cpp file?

Reply:
>Should I move the `#include` to the cpp file?

Yes. If it is required there, then it should be there.

Reply:
Done!